emulate-browsers/safari-13/index.ts

```diff
@@ -135,7 +137,14 @@ export default class Safari13 {
     const result = this.domOverrides.build();
     Object.assign(result[0], {
       callback: ({ cookie, origin }) => {
-        this.cookieJar.setCookie(cookie, origin);
+        try {
+          this.cookieJar.setCookie(cookie, origin);
+        } catch (error) {
+          log.warn('Error setting cookie from page', {
+            error,
+            sessionId: null, // TODO: @caleb - can you plug in sessionid to the emulators (or a child logger)
```

Comment:
left a @calebjclark todo for the next pr

Reply:
Where would I plug sessionId into the other emulators? There are no similar callbacks in the other emulator scripts.

Reply:
You would set the logger/sessionid on the emulator when it's constructed. So we'd be adding a property that all emulators would have injected